src/Simulation/CsharpGeneration.Tests/SimulationCodeTests.fs

```diff
@@ -3204,3 +3203,112 @@ namespace Microsoft.Quantum.Tests.LineNumbers
         |> 
         testOneFile (Path.Combine("Circuits","LineNumbers.qs"))
 
+        
+    [<Fact>]
+    let ``one file - UnitTests`` () =
+        """
```

Comment:
In hindsight, I think it would be a lot better if these strings, instead of a big string inside the .fs file would be all saved in their own file and during the test we just read the content of the file.

Reply:
That's a good idea. I made a task for it: https://dev.azure.com/ms-quantum/Quantum%20Program/_workitems/edit/9144

Reply:
This would be a fairly involved processes. Let's have that be in a separate PR.